src/fable-compiler-js/src/app.fs

```diff
@@ -2,6 +2,18 @@ module Fable.Compiler.App
 
 open Fable.Compiler.Platform
 open Fable.Compiler.ProjectParser
+open Fable.Core.JsInterop
+
+#if TEST_LOCAL
+let [<Emit("__dirname")>] __dirname: string = jsNative
```

Comment:
Compiling fable-compiler-js is failing in AppVeyor with

```
C:/projects/fable/src/fable-compiler-js/src/app.fs(8,6): (8,10) error FSHARP: The type 'Emit' is not defined.
C:/projects/fable/src/fable-compiler-js/src/app.fs(8,46): (8,54) error FSHARP: The value or constructor 'jsNative' is not defined.
```

It looks like an `open Fable.Core` is missing here.

Reply:
@alfonsogarciacaro Thanks, added.